File: Desarrollo/Wunallet/src/gestorAplicacion/infoClientes/Gestor.java
package gestorAplicacion.infoClientes;
import java.util.ArrayList;

public interface Gestor{
	public final float costoRomperTopes = (float)15000.0;	

	public abstract void sumarSaldo(float valorTransferencia);
	public abstract void restarSaldo(float valorTransferencia);
	
}

File: Python/gestorAplicacion/infoClientes/gestor.py
'''/Interface Gestor
Esta interface le exige a las clases que la implementan que declaren métodos de gestión de dichos productos financieros.
 Adicionalmente almacena constantes relevantes para la gestión de dichos productos.'''

from abc import ABC, abstractmethod

class Gestor(ABC):

    costoRomperTopes = 15000.0

    #Metodo para sumar saldo del producto bancario en que se implementa.
    @abstractmethod
    def sumarSaldo(self,valorTransferencia): pass

    #Metodo para restar saldo del producto bancario en que se implementa.

    @abstractmethod
    def restarSaldo(self,valorTransferencia): pass